gm4_chairs/data/gm4_chairs/loot_tables/drop_saddle.json

```diff
@@ -6,9 +6,15 @@
       "entries": [
         {
           "type": "minecraft:item",
-          "name": "minecraft:saddle"
+          "name": "minecraft:saddle",
+          "functions": [
+            {
+              "function": "minecraft:set_nbt",
+              "tag": "{Tags:[\"gm4_chairs_ignore\"]}"
```

Comment:
You can't apply entity NBT like this, the `set_nbt` function sets item NBT inside the `tag:{}` compound.

Reply:
Oops, I meant to remove that part from when I was testing as I didn't need it anymore